README.md

```diff
@@ -7,10 +7,10 @@ Digital samhandling mellom banker og eiendomsmeglere i forbindelse med gjennomf
 ## Beskrivelse av prosjektet
 Prosjektet er et samarbeid mellom Eiendom Norge, Finans Norge og Bits, samt deres medlemmer for å utarbeide bransjestandarder/spesifikasjoner for digital samhandling ved eiendomshandel.
 
-Prosjektleder: Monique Lier Nes (monique.nes@bits.no).
+Prosjektleder: Tonje Falch Lund (tonje.falch.lund@bits.no).
 
 ### Overordnede mål
-Muliggjøre en sikker og effektiv utveklsing av standardiserte og strukturerte data mellom bank-fagsystem og megler-fagsystem som er nødvendige for å gjennomføre elektronisk tinglysing.
+Muliggjøre en sikker og effektiv utveksling av standardiserte og strukturerte data mellom bank-fagsystem og megler-fagsystem som er nødvendige for å gjennomføre elektronisk tinglysing.
```

Comment:
Kommer vel kanskje en del meldingstyper som effektiviserer ting uavhengig av om det skal e-tinglysing eller ikke også forøvrig? Kjøpekontrakt, etc f.eks. Ikke sikkert det nødvendigvis trenger å stå så mye mer om dette forøvrig, bare noe jeg kom til å tenke på :)

Reply:
Godt poeng, kan oppdatere hele avsnittet til dette (deler tatt ut av mandat): Å etablere en heldigital samhandling mellom bank og meglerforetak. Dette inkluderer kommunikasjon og utveksling av informasjon, elektronisk tinglysing av skjøtepakker og oversendelse av ulike dokumenter.